blocks/cards/cards.css

```diff
@@ -149,3 +186,245 @@
+  .cards.info-card .card-container {
+    padding-left: 20px;
+    padding-right: 20px;
+    cursor: pointer;
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    flex-direction: column;
+    height: 100%;
+    -webkit-box-pack: end;
+    -ms-flex-pack: end;
+    justify-content: flex-end;
+    min-height: 31.9rem;
+    position: relative;
+  }
+
+  /* <a> tag body of the card to take the background and make the full card clickable
+    Went with that over a clickable div element to match the client structure since it seemed pretty clean */
+  .cards.info-card .cards-card-body {
+    background-color: var(--info-card-background-color);
+    margin: 0;
+    position: relative;
+    box-sizing: border-box;
+    text-decoration: none;
+    color: var(--info-card-color);
+    cursor: pointer;
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    flex-direction: column;
+    -webkit-box-pack: end;
+    -ms-flex-pack: end;
+    justify-content: flex-end;
+    min-height: 31.9rem;
+    padding: 4rem;
+  }
+
+  /* Display the pretty gradient on hover on the card */
+  .cards.info-card .card-container:hover .cards-card-body {
+    background: linear-gradient(140deg, var(--info-card-gradient-background-color-start) 24%, var(--info-card-gradient-background-color-end));
+  }
+
+  /* Layout and sizing for the title of the card */
+  .cards.info-card .header-container {
+    margin-bottom: 2rem;
+    -webkit-box-align: end;
+    -ms-flex-align: end;
+    align-items: flex-end;
+    display: -webkit-box;
+    display: flex;
+  }
+
+/* Font styling for the title of the card */
+.cards.info-card .card-title {
+  font-size: 2.3rem;
+  font-weight: 500;
+  letter-spacing: -.12rem;
+  line-height: 3.2rem;
+  margin-bottom: 2rem;
+  color: var(--info-card-color);
+  text-decoration: none;
+}
+
+  /* Layout and styling of the button in the card */
+  .cards.info-card .button.primary {
+    background-color: transparent;
+    border: 0;
+    border-radius: 0;
+    padding: 0;
+    white-space: normal;
+    margin-top:0;
+    margin-bottom: 3rem;
+    color: var(--info-card-color);
+  }
+
+  /* Text styling of the button in the card */
+  .cards.info-card .button-container {
+    font-size: 1.4rem;
+    font-weight: 400;
+    letter-spacing: -.02rem;
+    line-height: 2rem;
+    text-decoration: none;
+    text-align: left;
+  }
+
+  /* Styling of the arrow icon on the card */
+  .cards.info-card .icon-arrow-link {
+    font-size: 1.7rem;
+    letter-spacing: -.025rem;
+    line-height: 2.5rem;
+    font-weight: 400;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+  }
+
+  /* Styling of the arrow icon and it's animation */
+  .cards.info-card .icon-arrow-link::before {
+    content: var(--vlt-right-arrow);
+    color: var(--info-card-color);
+    font-family: VltIcons!important;
```

Comment:
there is now a variable for this font

Reply:
Groovy!  Will find and update my instances using it

Reply:
Whelp I added my own spin on the variable to this branch (given that the one you mentioned is not in main yet), if you get your PR merged in first I'll rebase